src/dune_pkg/fetch.ml

```diff
@@ -90,13 +105,10 @@ let fetch ~checksum ~target url =
                    ("checksum", `String (Checksum.to_string checksum)) :: args))
           }))
   in
-  (* hashes have to be empty otherwise OPAM deletes the file after
```

Comment:
Where did this comment go?

Reply:
Upward into where it constructs the `pull` closure. It actually applies to both branches so I pulled it into a single identifier with the comment in 52548b92e.